libs/common/src/schemas/deal-cancellation.ts

```diff
@@ -0,0 +1,11 @@
+import z from 'zod';
+import { TfmDealCancellation, ZodTypeFromObject } from '../types';
+import { MAX_CHARACTER_COUNT } from '../constants';
+
+export const DEAL_CANCELLATION: z.ZodObject<ZodTypeFromObject<TfmDealCancellation>> = z
```

Comment:
Documentation?

Can we also use `export default` as there's only 1x function exported

Reply:
I wasn't aware export default is the prefered.

I've got a slight preference against it - its not extensible & it does slightly weird things if you import into a common js file & you have to write 
```
const { default: theDefaultExport } = require('../path/to/file');
```
Happy to do if this is the pattern we want to run with

Reply:
Not a blocker thanks @MarRobSoftwire I think its subjective, pros and cons.

Personally I think should always a default export if there's only one thing, I believe it's the default linting rule and we have it disabled

But we already have a mix of both approaches so happy to resolve

Reply:
Just some documentation on this function though please 🙏

Reply:
Have added :)